fidesctl/src/fidesctl/cli/__init__.py

```diff
@@ -35,6 +35,9 @@
     status,
 ]
 ALL_COMMANDS = API_COMMANDS + LOCAL_COMMANDS
+SERVER_CHECK_COMMAND_NAMES = [
+    command.name for command in API_COMMANDS if command.name != "status"
+]
```

Comment:
This is probably the most Python block of code I've ever seen. By that I mean: both exceedingly efficient and confusing as all hell. How about:
```suggestion
SERVER_CHECK_COMMAND_NAMES = [
    command.name for command in API_COMMANDS
].remove("status")
```
It's a subtle change but I think it makes a ton of difference in readability.

Reply:
Actually I don't think that suggestion works as-is. I don't think `.remove()` returns the list. It might need to happen in two steps instead.

Reply:
Thanks @PSalant726 ! I'll run it through quick and see how your suggestion plays out, I think there is probably a way to improve readability like you are looking for in case it doesn't work 🤷🏽 

In regard to the long-term worries, I think you may be right... However, I'm hopeful the planned changes for tracking `init` may give us some added wiggle room and was hoping to use this to at least unblock the PRs with failing checks. today

Reply:
Think I made the change you suggested above @PSalant726 (two steps), re-requesting a review - thanks again!

Reply:
I'm sorry I had to swoop in and combine both of your ideas. modifying a list in place is a _not in my backyard_ thing for me, but i think by saying `not in` and using a list we can actually make this more readable and easy to extend if other edge-cases pop up 😄 

thank you both for the input ✋